boards/xtensa/intel_adsp_cavs15/tools/flash.sh

```diff
@@ -5,15 +5,16 @@
 BUILD=$1
 FIRMWARE=${BUILD}/zephyr/zephyr.ri
 FLASHER=${ZEPHYR_BASE}/boards/xtensa/intel_adsp_cavs15/tools/fw_loader.py
+RIMAGE=${ZEPHYR_BASE}/../modules/audio/sof/zephyr/ext/rimage/build/rimage
```

Comment:
Not sure how this script is called and when, it will be safer if you just define an environment variable pointing to the location of rimage (whether it is locally installed, or part of sof tree or the sof module) and then use it in this script and keep it flexible and simple. Or you could enhance the script to accept more arguments, including the rimage location.

Reply:
OK, I will add more arguments

Reply:
This script is called by west flash or when we use twister to run multiple tests.